### DISCUSSION TITLE
Is there a way to not create the .zshrc file?

### DISCUSSION THREAD

**Discussion by mrsauravsahu** [ohmyzsh/ohmyzsh#9970] [Help] [answered]
I'm using the installer in one of my scripts and I want to make sure the installation doesn't make any changes outside the install directory.

ohmyzsh currently backs up any .zshrc files in the home directory or creates one if not present. Can I stop this behavior?

This is how I'm installing ohmyzsh in a specific directory - https://github.com/mrsauravsahu/cli-config/blob/cool/scripts/install.ohmyzsh.sh#L4

**Comment by mcornella** [comment, marked as answer]
You can run the installer with a `--keep-zshrc` argument. You can pass the argument by running it as the README says:
```sh
sh -c "$(curl -fsSL https://raw.githubusercontent.com/ohmyzsh/ohmyzsh/master/tools/install.sh)" "" --keep-zshrc
```

**Reply by mrsauravsahu** [reply]
@mcornella Thanks this worked great. Could you point me to the README? I saw it written inside the script, I was looking all over on Github for this though

**Reply by jeffmhubbard** [reply]
The README is what is displayed below the file list @ https://github.com/ohmyzsh/ohmyzsh. It's also installed with `ohmyzsh` as `~/.oh-my-zsh/README.md`

**Reply by mrsauravsahu** [reply]
Yeah, the options were inside the script itself - got what I was looking for here - https://github.com/ohmyzsh/ohmyzsh/blob/36f444ed7325720ec05f182781ec7d6c9a4d675c/tools/install.sh#L27 @jeffmhubbard
